From 5aedebb7282075dd2283d90d4a0741ba4f1bc716 Mon Sep 17 00:00:00 2001
From: K9100ii <k9100ii.ki@gmail.com>
Date: Thu, 19 May 2022 00:27:47 +0100
Subject: [PATCH 1/2] Revert "SystemUI: Use AVCProfileMain for screen recorder"

This reverts commit 639443f6db76733d9f0d65d937d928b3cbb08d25.

On devices where the High AVC profile works fine, screen recordings are
better quality with it.

Change-Id: I31acf97f17582ecb55788d6c462f40aefad5dcd0
---
 .../com/android/systemui/screenrecord/ScreenMediaRecorder.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/screenrecord/ScreenMediaRecorder.java b/packages/SystemUI/src/com/android/systemui/screenrecord/ScreenMediaRecorder.java
index cfda8be5a270..b8d96f774e02 100644
--- a/packages/SystemUI/src/com/android/systemui/screenrecord/ScreenMediaRecorder.java
+++ b/packages/SystemUI/src/com/android/systemui/screenrecord/ScreenMediaRecorder.java
@@ -148,7 +148,7 @@ public class ScreenMediaRecorder extends MediaProjection.Callback {
                 * VIDEO_FRAME_RATE_TO_RESOLUTION_RATIO;
         mMediaRecorder.setVideoEncoder(MediaRecorder.VideoEncoder.H264);
         mMediaRecorder.setVideoEncodingProfileLevel(
-                MediaCodecInfo.CodecProfileLevel.AVCProfileMain,
+                MediaCodecInfo.CodecProfileLevel.AVCProfileHigh,
                 MediaCodecInfo.CodecProfileLevel.AVCLevel3);
         mMediaRecorder.setVideoSize(width, height);
         mMediaRecorder.setVideoFrameRate(refreshRate);
-- 
2.48.1

